storage/id_alloc_test.go

```diff
@@ -118,6 +119,7 @@ func TestNewIDAllocatorInvalidArgs(t *testing.T) {
 // 4) Make IDAllocator valid again, the blocked allocations return correct ID.
 // 5) Check if the following allocations return correctly.
 func TestAllocateErrorAndRecovery(t *testing.T) {
+	defer leaktest.AfterTest(t)
 	store, _ := createTestStore(t)
 	allocd := make(chan int, 10)
```

Comment:
Hey Joe, don't worry about it too much - I'll take a look later. Do you really need the 50ms of sleep?